tasks/fill-nonce-gap.ts

```diff
@@ -7,11 +7,11 @@ async function trySend(
   gasPrice: number,
 ) {
   try {
-    const tx = await signer.sendTransaction({ to: address, value: 0, nonce: nonce, gasPrice: gasPrice });
+    const tx = await signer.sendTransaction({ to: address, value: 0, nonce: nonce, gasPrice: gasPrice * 1000000000 });
```

Comment:
Not sure, why do we multiply gas price by `1000000000`?

Reply:
number of wei in 1eth